var/spack/repos/builtin/packages/exawind/package.py

```diff
@@ -6,35 +6,118 @@
 
-    maintainers("jrood-nrel", "psakievich")
+    maintainers = ["jrood-nrel"]
 
     tags = ["ecp", "ecp-apps"]
 
-    version("master", branch="main")
+    # Testing is currently always enabled, but should be optional in the future
+    # to avoid cloning the mesh submodule
+    version("master", branch="main", submodules=True)
 
-    depends_on("trilinos+stk")
-    depends_on("tioga+shared~nodegid")
-    depends_on("nalu-wind+hypre+openfast+tioga+wind-utils")
-    depends_on("amr-wind+hypre+mpi+netcdf+openfast")
-    depends_on("openfast+cxx+shared@2.6.0:")
+    variant("openfast", default=False, description="Enable OpenFAST integration")
+    variant("hypre", default=True, description="Enable hypre solver")
+    variant("stk_simd", default=False, description="Enable SIMD in STK")
+    variant("umpire", default=False, description="Enable Umpire")
+    variant("tiny_profile", default=False, description="Turn on AMR-wind with tiny profile")
+    variant("sycl", default=False, description="Enable SYCL backend for AMR-Wind")
+    variant("gpu-aware-mpi", default=False, description="gpu-aware-mpi")
+
+    conflicts("+hypre", when="+sycl")
+
+    for arch in CudaPackage.cuda_arch_values:
+        depends_on(
+            "amr-wind+cuda cuda_arch=%s" % arch, when="+amr_wind_gpu+cuda cuda_arch=%s" % arch
+        )
+        depends_on(
+            "nalu-wind+cuda cuda_arch=%s" % arch, when="+nalu_wind_gpu+cuda cuda_arch=%s" % arch
+        )
+        depends_on(
+            "trilinos+cuda cuda_arch=%s" % arch, when="+nalu_wind_gpu+cuda cuda_arch=%s" % arch
+        )
+
+    for arch in ROCmPackage.amdgpu_targets:
+        depends_on(
+            "amr-wind+rocm amdgpu_target=%s" % arch,
+            when="+amr_wind_gpu+rocm amdgpu_target=%s" % arch,
+        )
+        depends_on(
+            "nalu-wind+rocm amdgpu_target=%s" % arch,
+            when="+nalu_wind_gpu+rocm amdgpu_target=%s" % arch,
+        )
+        depends_on(
+            "trilinos+rocm amdgpu_target=%s" % arch,
+            when="+nalu_wind_gpu+rocm amdgpu_target=%s" % arch,
+        )
+
+    depends_on("nalu-wind+tioga")
+    depends_on("amr-wind+netcdf+mpi")
+    depends_on("tioga~nodegid")
     depends_on("yaml-cpp@0.6:")
+    depends_on("nalu-wind+openfast", when="+openfast")
+    depends_on("openfast+cxx@2.6.0:", when="+openfast")
+    depends_on("openfast+cxx@2.6.0:", when="^nalu-wind+openfast")
+    depends_on("openfast+cxx@2.6.0:", when="^amr-wind+openfast")
+    depends_on("amr-wind+hypre", when="+hypre")
+    depends_on("amr-wind~hypre", when="~hypre")
+    depends_on("nalu-wind~hypre", when="~hypre")
+    depends_on("trilinos+ninja", when="+ninja")
+    depends_on("nalu-wind+ninja", when="+ninja")
+    depends_on("amr-wind+ninja", when="+ninja")
+    depends_on("amr-wind+sycl", when="+sycl")
+    # not required but added so these get picked up as a
+    # direct dependency when creating snapshots
+    depends_on("trilinos")
```

Comment:
Should we remove these direct dependencies since they aren't necessary?

Reply:
sure thing